### PR TITLE
New landscape images

### PR DESCRIPTION
**Pull request by jafdy**
#2015 
The product of the recently closed contest I ran on the steam forums has given me a lot of good landscape images.
The contest thread is [here](http://steamcommunity.com/app/404410/discussions/3/340412122420889457/).
Google Drive with all images [here](https://drive.google.com/open?id=0B0mWx-Fhhhz7WkxLemZNeGFLbUE)

### CODE REVIEW

**Comment by jafdy** [comment]
@endless-sky do you need me to do anything with these pictures before you can add them?

**Comment by Amazinite** [comment]
I'd think putting them all into the copyright.txt and finding planets to put them on would help.

**Comment by Amazinite** [comment]
There are currently 242 landscape images, meaning this would take that to a total of 355. There are only 323 planets in the game (got that number counting how many times "landscape" shows up in map.txt). That means that not only should there be no repeats, but there will be extra images left over. xD

**Comment by endless-sky** [comment]
Yeah, I'm a little hesitant to add this many pictures all at once. And some of these are really great, but a lot of them have issues - blown-out highlights, haziness or fuzziness, weird framing (e.g. mountains going all the way off the top of the image), signs of human habitation that might look anachronistic in a sci-fi game, etc.

**Comment by Lineth** [comment]
For the human habitation issue, doesn't the game already have a lot of images like that?

**Comment by jafdy** [comment]
I'll get on with sorting the copyright. 
I never expected all of them to get in but I have included all the images that I thought where good enough. If you don't like any of them I can remove them (heck, you could remove them) quite easily enough.
I am reluctant to start matching them to planets before I know what ones are being kept.

File: credits.txt [hunk 1]
```diff
@@ -11,6 +11,8 @@ Photos from Wikimedia Commons
   Matthew Field (CC-BY-SA 3.0)
 and other photographers
   Chris Pickel (CC-BY-SA 4.0)
+  Michael Wilson (CC-BY-4.0)
+  @Gardevou (CC-BY-4.0)
```
Comment (KiLEdEnNis) [inline credits.txt]: isn't it Gardevour? (also in the fields17.jpg & copyright) it is on Discord so that's why

Comment (jafdy) [inline credits.txt]: It's the name emotion wants to go under. I can't make him use his real name. (Yes, it is the same as the one from discord.

**Comment by jafdy** [comment]
That's the copyright done. I still don't know what ones you want to keep. I'm not going to be very good at narrowing them down further.

**Comment by jafdy** [comment]
@endless-sky I can do anything that you need me to but I can't pick out all the ones you don't like if you don't say anything.

**Comment by endless-sky** [comment]
I would vote for including the following, and saving the others as a library of landscapes for content creators to use if they need more variety. These are ones that I think are sufficiently different from what's already there, are free from any obvious visual flaws, and present the most degree of drama or interest:

badlands7
beach12
beach19
canyon11
desert11
dune6
hills9
mountain16
mountain22
mountain24
mountain28
mountain30
mountain31
sea16
sea25
sea28
valley11
valley12
valley16
valley22
valley26
water17
water21

**Comment by jafdy** [comment]
Umm, badlands 7 is one that was already in the game. That said I did forget to copyright badlands 9.

**Comment by endless-sky** [comment]
I was going by the names the images had in your Google Drive folder; "badlands7" was this one:

badlands7

Also, would it be possible to license these CC-BY-SA-4.0 rather than CC-BY-4.0? The "share-alike" license is more in keeping with open source, because it says that if someone uses the images from the game to create something new, that new creation should be freely shared, too. The image authors are free to license the images in other terms in other contexts, of course, because they maintain the copyright to them.

**Comment by jafdy** [comment]
It was meant to 17 not 7. it must have slipped past before I uploaded to google drive.

**Comment by DingusShingleton** [comment]
I would just like to say that this is a perfect example of now ES and its development leaves me in awe. From concept to execution to incorporation into the game, this image contest is just awesome. Bravo @jafdy.

**Comment by jafdy** [comment]
*bows*

**Pull request merged**